src/drtsans/mono/biosans/api.py

```diff
@@ -1488,7 +1493,7 @@ def _prepare_sample_transmission_ws(_sample_transmission):
                         "background": None,
                     },
                 )
-        except ZeroMonitorCountsError as e:
+        except (ZeroMonitorCountsError, TransmissionErrorToleranceError, TransmissionNanError) as e:
```

Comment:
as far as I understand, the `TransmissionErrorToleranceError` and `TransmissionNanError` exceptions can only arise when evaluating:
```
                _, sample_trans_ws = _prepare_sample_transmission_ws(raw_sample_ws)
```
Thus, a new  related `try/except` block should encapsulate it:
```
try:
    _, sample_trans_ws = _prepare_sample_transmission_ws(raw_sample_ws)
except (TransmissionErrorToleranceError, TransmissionNanError) as e:
    logger.warning(f"Skipping slice {sample_name}: {e}.")
    continue
```